schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/activity/DefaultActivityService.java

```diff
@@ -455,11 +398,8 @@ public Activity updateActivity(Activity activity) throws EntityNotFoundException
     }
 
     @Override
-    public void removeActivity(Key id) throws EntityNotFoundException, IllegalArgumentException, OperationException {
```

Comment:
Here also I have my doubts...  Specifically for the removal we are forcing an extra, uncecessary read of whoever is deleting it.  I would like to have `ActivityDao.remove(id);` only and let the controller be responsible for guaranteeing that the id is ok.  For example, deleting an org:
```java
keys = OrgDao.getActivityKeys();
foreach key in keys ActivityDao.remove(key);
```